addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorMagnetHandler.java

```diff
@@ -52,6 +65,10 @@ void parseReport(JsonObject data) {
                 }
             }
         }
+        if (data.has(VOLTAGE)) {
```

Comment:
does the aqara sensor report the voltage in the report message? I've only seen them in heartbeat messages so far...